source/systemUtils.py

```diff
@@ -11,14 +11,17 @@
 import winUser
 import os
 
-
 def openUserConfigurationDirectory():
 	"""Opens directory containing config files for the current user"""
-	import config
-	path = config.getUserDefaultConfigPath()
-	if not path:
-		raise ValueError("no user default config path")
-	config.initConfigPath(path)
+	import globalVars
+	try:
+		path = globalVars.appArgs.configPath
```

Comment:
When do you expect to get an `AttributeError`?

Looking at `source/core.py` line 222, I think you can rely on `appArgs.configPath` being set correctly.

Reply:
When NVDA is not running and this function is executed by NVDA_slave

Reply:
Ok. Can you add a comment to point that out. Something like:
```suggestion
		# configPath is guaranteed to be correct for NVDA, however it will not exist for NVDA_slave.
		path = globalVars.appArgs.configPath
```